tests/contrib/patch.py

```diff
@@ -0,0 +1,119 @@
+import sys
+
+import wrapt
+
+
+class PatchMixin(object):
```

Comment:
I wonder if we should separate the additional `PatchMixin` helpers from the required test cases.

e.g. 

```python
class PatchMixin(unittest.TestCase):
    def module_imported(self, modname): pass


class BasePatchTestCase(BaseTracerTestCase, PatchMixin):
    def test_patch(self): raise NotImplemented()
```

Reply:
We should probably have `PatchMixin` inherit from `unittest.TestCase`, since it uses some of those methods itself?

Reply:
Yeah makes sense to separate in case the utilities are needed in a non-patch test-case scenario.